### ISSUE TITLE
Exported client-side scripts are inconsistent

### ISSUE THREAD

**Issue by jefflembeck** [npm/newww#1333]
We don't have a set way that we like to write our front-end scripts, and they all just end up in the same folder. I'd like to add some structure/ "a way that we do things", for easier maintenance/code reviews.

**Comment by aredridel** [comment]
Indeed. I think this will fall out of a more structured concept for how frontend work is done, and how we make components.
